pkg/cmd/pulumi/crypto_http_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/pulumi/pulumi/sdk/v2/go/common/resource/config"
+	"github.com/pulumi/pulumi/sdk/v2/go/common/workspace"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestChangeProjectStackSecretDetails(t *testing.T) {
+	//projStack := &workspace.ProjectStack{
+	//	Name:    tokens.PackageName("test-package"),
+	//	Runtime: workspace.NewProjectRuntimeInfo("nodejs", nil),
+	//}
+	//
+	//assert.Equal(t, "foo", prettyKeyForProject(config.MustMakeKey("test-package", "foo"), proj))
+	//assert.Equal(t, "other-package:bar", prettyKeyForProject(config.MustMakeKey("other-package", "bar"), proj))
```

Comment:
Why are these lines commented out? Should they be deleted?

Reply:
Yes, they are now gone!